jdbc-adapter/virtualschema-jdbc-adapter/src/main/java/com/exasol/adapter/jdbc/JdbcAdapterProperties.java

```diff
@@ -43,6 +44,7 @@
     static final String PROP_EXCLUDED_CAPABILITIES = "EXCLUDED_CAPABILITIES";
     static final String PROP_EXCEPTION_HANDLING = "EXCEPTION_HANDLING";
     static final String PROP_LOG_LEVEL = "LOG_LEVEL";
+    static final String PROP_IGNORE_ERROR_LIST = "IGNORE_ERROR_LIST";
```

Comment:
The EXCEPTION_HANDLING property just swallows exceptions by the remote system, we do s.th. different here.
We go with:
`IGNORE_ERRORS='POSTGRES_UPPERCASE_TABLES'`
instead of the name you can also specify the error number that is thrown.